corporate/lib/decorator.py

```diff
@@ -43,19 +43,19 @@ def _wrapped_view_func(
 
 def authenticated_remote_realm_management_endpoint(
     view_func: Callable[Concatenate[HttpRequest, RemoteRealmBillingSession, ParamT], HttpResponse]
-) -> Callable[Concatenate[HttpRequest, ParamT], HttpResponse]:  # nocoverage
+) -> Callable[Concatenate[HttpRequest, ParamT], HttpResponse]:
     @wraps(view_func)
     def _wrapped_view_func(
         request: HttpRequest,
         /,
         *args: ParamT.args,
         **kwargs: ParamT.kwargs,
     ) -> HttpResponse:
-        if not is_self_hosting_management_subdomain(request):
+        if not is_self_hosting_management_subdomain(request):  # nocoverage
             return render(request, "404.html", status=404)
```

Comment:
Probably we should just add a little test for this to get to 100% coverage on this file.